tests/src/JIT/HardwareIntrinsics/X86/Avx/Add.Double.cs

```diff
@@ -77,26 +86,30 @@ private static void AddDouble()
     public sealed unsafe class SimpleBinaryOpTest__AddDouble
     {
         private const int VectorSize = 32;
-        private const int ElementCount = VectorSize / sizeof(Double);
 
-        private static Double[] _data1 = new Double[ElementCount];
-        private static Double[] _data2 = new Double[ElementCount];
+        private const int Op1ElementCount = VectorSize / sizeof(Double);
```

Comment:
We track the element count of each operand and the return value separately to give more control when they differ.